CHANGELOG.md

```diff
@@ -6,16 +6,21 @@
   * Adding new optional parameter (indicating the columns attr to clean filters) to the `clearColumnFilters` method. ([#628](https://github.com/OntimizeWeb/ontimize-web-ngx/issues/628)) ([c413ece](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/c413ece))
   * Adding new optional parameter (indicating the new sort columns) to the `reinitializeSortColumns` method and changing its access level to public.([#628](https://github.com/OntimizeWeb/ontimize-web-ngx/issues/628)) ([c413ece](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/c413ece))
   * **o-table-cell-editor-email**: new component for email cell editors([6fdf754](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/6fdf754))
-  * Adding virtual scroll ([#598](https://github.com/OntimizeWeb/ontimize-web-ngx/issues/598)) ([e14c424](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/e14c424) ([238da22](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/238da22) ([b5668c3](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/b5668c3)
+  * Adding virtual scroll ([#598](https://github.com/OntimizeWeb/ontimize-web-ngx/issues/598)) ([e14c424](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/e14c424) ([238da22](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/238da22)) ([b5668c3](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/b5668c3)) ([4cabf1a](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/4cabf1a)) ([59e7684](https://github.com/OntimizeWeb/ontimize-web-ngx/commit/59e7684)) 
   
-    *NOTE*: Set min-height='400px' in o-table, you can modify the height table in .o-table class
+    *NOTE*
+      * The virtual scrolling and the table tooltip are incompatible, if you want the tooltip functionality on the table, you must deactivate virtual scrolling as follows `virtual-scroll = no` because it is actived by default
```

Comment:
```suggestion
      * The virtual scrolling and the table tooltip are incompatible due to a recognized bug in Angular library, if you want the tooltip functionality on the table, you must deactivate virtual scrolling as follows `virtual-scroll = no` because it is actived by default
```